scipy/spatial/_spherical_voronoi.py

```diff
@@ -139,14 +140,16 @@ def __init__(self, points, radius=1, center=None, threshold=1e-06):
                           DeprecationWarning)
 
         self.points = points
-        self.radius = radius
-        self.threshold = threshold
-        self._dim = len(points[0])
+        self.radius: float = radius
```

Comment:
The annotations here should be unnecessary-the types will be inferred from the types in the signature.

Reply:
I've removed them